python/cuml/fil/fil.pyx

```diff
@@ -348,21 +359,27 @@ cdef class ForestInference_impl():
                                  str algo,
                                  float threshold,
                                  str storage_type,
-                                 int blocks_per_sm):
+                                 int blocks_per_sm,
+                                 int threads_per_tree,
+                                 int n_items):
         TreeliteQueryNumClass(<ModelHandle> model.handle,
                               & self.num_class)
         return self.load_from_treelite_model_handle(<uintptr_t>model.handle,
                                                     output_class, algo,
                                                     threshold, storage_type,
-                                                    blocks_per_sm)
+                                                    blocks_per_sm,
+                                                    threads_per_tree,
+                                                    n_items)
 
     def load_using_treelite_handle(self,
                                    model_handle,
                                    bool output_class,
                                    str algo,
                                    float threshold,
                                    str storage_type,
-                                   int blocks_per_sm):
+                                   int blocks_per_sm,
```

Comment:
Not for this pull request: I wonder if this can be refactored into a list of keyword arguments `**kwargs`, to avoid threading each new parameter manually.